src/Models/Setting.php

```diff
@@ -2,17 +2,35 @@
 
 namespace Portable\FilaCms\Models;
 
+use Illuminate\Database\Eloquent\Attributes\ObservedBy;
 use Illuminate\Database\Eloquent\Model;
+use Illuminate\Support\Facades\Cache;
+use Portable\FilaCms\Observers\SettingObserver;
 
+#[ObservedBy(SettingObserver::class)]
 class Setting extends Model
 {
+    protected string $cacheKey;
+
     protected $fillable = [
         'key',
         'value'
     ];
 
+    public function __construct(array $attributes = [])
+    {
+        parent::__construct($attributes);
+
+        $this->cacheKey = 'setting-' . $this->key;
```

Comment:
I think you should make cacheKey an attribute.

If I do this:
````$setting = new Setting();
$setting->key = 'mything';```

Then `$setting->cacheKey` will be `setting-null`